# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1309406866 25200
#      Wed Jun 29 21:07:46 2011 -0700
# Branch stable
# Node ID 944cf42c699e32978009df6ab1baea9284795b8c
# Parent  d02c9a58bae1a4634ef8cf7610d3fad0598dbca3
doc: small documentation fixes for strftime, betainc

betainc.cc: Place variable x inside @var macro
strftime.cc: Place '%' escape character before '%','n','t' commands.

diff --git a/src/DLD-FUNCTIONS/betainc.cc b/src/DLD-FUNCTIONS/betainc.cc
--- a/src/DLD-FUNCTIONS/betainc.cc
+++ b/src/DLD-FUNCTIONS/betainc.cc
@@ -52,17 +52,17 @@ Return the regularized incomplete Beta f
 betainc (x, a, b) = -----------    | t^(a-1) (1-t)^(b-1) dt.\n\
                      beta (a, b)    /\n\
                                  t=0\n\
 @end group\n\
 @end smallexample\n\
 \n\
 @end ifnottex\n\
 \n\
-If x has more than one component, both @var{a} and @var{b} must be\n\
+If @var{x} has more than one component, both @var{a} and @var{b} must be\n\
 scalars.  If @var{x} is a scalar, @var{a} and @var{b} must be of\n\
 compatible dimensions.\n\
 @end deftypefn")
 {
   octave_value retval;
 
   int nargin = args.length ();
 
diff --git a/src/DLD-FUNCTIONS/time.cc b/src/DLD-FUNCTIONS/time.cc
--- a/src/DLD-FUNCTIONS/time.cc
+++ b/src/DLD-FUNCTIONS/time.cc
@@ -335,23 +335,23 @@ strftime (\"%r (%Z) %A %e %B %Y\", local
 \n\
 Octave's @code{strftime} function supports a superset of the ANSI C\n\
 field specifiers.\n\
 \n\
 @noindent\n\
 Literal character fields:\n\
 \n\
 @table @code\n\
-@item %\n\
+@item %%\n\
 % character.\n\
 \n\
-@item n\n\
+@item %n\n\
 Newline character.\n\
 \n\
-@item t\n\
+@item %t\n\
 Tab character.\n\
 @end table\n\
 \n\
 @noindent\n\
 Numeric modifiers (a nonstandard extension):\n\
 \n\
 @table @code\n\
 @item - (dash)\n\
